opentelemetry-api/src/opentelemetry/baggage/propagation/__init__.py

```diff
@@ -54,27 +63,30 @@ def extract(
         baggage_entries = header.split(",")
         total_baggage_entries = self._MAX_PAIRS
         for entry in baggage_entries:
+
             if total_baggage_entries <= 0:
-                return context
+                return context  # type: ignore
             total_baggage_entries -= 1
             if len(entry) > self._MAX_PAIR_LENGTH:
+                _logger.warning("Entry %s has been discarded", entry)
                 continue
             try:
                 name, value = entry.split("=", 1)
             except Exception:  # pylint: disable=broad-except
+                _logger.warning("Entry %s has been discarded", entry)
```

Comment:
Why are we dropping extra members, invalid member? Shouldn't we return the original context if we find baggage string corrupted. We do this in trace context trace state parse.